test/nodes/test_connector.py

```diff
@@ -161,3 +168,17 @@ def test_crawler_extract_hidden_text(test_url, tmp_path):
     )
     crawled_content = documents["documents"][0].content
     assert "hidden text" not in crawled_content
+
+
+def test_crawler_loading_wait_time(test_url, tmp_path):
+    loading_wait_time = 3
+    crawler = Crawler(output_dir=tmp_path)
+    paths = crawler.crawl(urls=[test_url + "/page_dynamic.html"], crawler_depth=1, loading_wait_time=loading_wait_time)
+
+    assert len(paths) == 4
+    assert content_in_results(
+        crawler, test_url + "/page_dynamic_result.html", paths, loading_wait_time=loading_wait_time
```

Comment:
As I mentioned above, I think here the comparison would work even without the `loading_wait_time` parameter